apps/backend/src/miscellaneous.rs

```diff
@@ -125,6 +81,49 @@ use crate::{
         user_id_from_token, AUTHOR, SHOW_SPECIAL_SEASON_NAMES, TEMP_DIR, VERSION,
     },
 };
+use anyhow::Result as AnyhowResult;
```

Comment:
Any reason why you move third party imports below crate ones? I tend to keep them above.